### PR TITLE
modal signin signup

### PR DESCRIPTION
**Pull request by bapboy18**

### CODE REVIEW

File: app/views/layouts/_signin.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<div id="myModal" class="modal fade" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal">&times;</button>
```
Comment (daothanhcam) [inline app/views/layouts/_signin.html.erb]: Break line.

File: app/views/shared/_button.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<div id="button">
+  <h4>
+    <strong>
+      <%= link_to t("signin"),"#",
+        data: {toggle: :modal, target: "#myModal",
+          remote: new_user_session_url}, onclick: "onHrefClick();" %>
```
Comment (daothanhcam) [inline app/views/shared/_button.html.erb]: Indent.

File: app/views/shared/_button.html.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    </strong>
+    <strong>
+      <%= link_to t("signup"), "#",
+        data: {toggle: :modal, target: "#myModal",
+          remote: new_user_registration_url}, onclick: "onHrefClick();" %>
```
Comment (daothanhcam) [inline app/views/shared/_button.html.erb]: Indent.

File: app/views/layouts/_signin.html.erb [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal">&times;
+        </button>
+        <h4 class="modal-title"></h4>
+      </div>
+      <div class="modal-body">
+      </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-default" data-dismiss="modal">
+        </button>
```
Comment (ThuBM) [inline app/views/layouts/_signin.html.erb]: <button ... />

File: app/views/shared/_button.html.erb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<div id="button">
+  <h4>
+    <strong>
+      <%= link_to t("signin"),"#",
```
Comment (ThuBM) [inline app/views/shared/_button.html.erb]: add space before "#"

File: app/views/shared/_button.html.erb [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+        data: {toggle: :modal, target: "#myModal",
+        remote: new_user_session_url}, onclick: "onHrefClick();" %>
```
Comment (ThuBM) [inline app/views/shared/_button.html.erb]: shouldn't write onclick onclick: onHrefClick()

File: app/views/shared/_button.html.erb [hunk 7]
```diff
@@ -0,0 +1,14 @@
...
+    </strong>
+    <strong>
+      <%= link_to t("signup"), "#",
+        data: {toggle: :modal, target: "#myModal",
+        remote: new_user_registration_url}, onclick: "onHrefClick();" %>
```
Comment (ThuBM) [inline app/views/shared/_button.html.erb]: shouldn't write onclick: onHrefClick()

File: app/views/layouts/_signin.html.erb (hunk 4 above)
Comment (bapboy18) [inline app/views/layouts/_signin.html.erb]: what is comment?

**Pull request merged**